### ISSUE TITLE
Adopt/reuse rustracing?

### ISSUE THREAD

**Issue by rbtcollins** [opentracing/opentracing-rust#9]
Have y'all seen rustracing already? Might be a big timesaver to leverage that implementation which has been around for a few months. https://github.com/sile/rustracing and https://github.com/sile/rustracing_jaeger for the jaeger bindings.

**Comment by daschl** [comment]
@rbtcollins yes, I've been in contact upfront with every opentracing impl maintainer I could find on crates.io via email. Unfortunately he doesn't have time to help out with our efforts at the moment, but its definitely something we should look at going forward.

It probbably makes sense to keep the old/alternative implementations in a wiki for now.
